internal/ui/main_page.go

```diff
@@ -0,0 +1,53 @@
+package ui
+
+import (
+	"github.com/rivo/tview"
+)
+
+func MainPage(app *tview.Application) error {
+	layoutListDownload := tview.NewFlex().
+		SetDirection(tview.FlexRow)
+
+	txtDownloads := tview.NewTextView().
+		SetText("Downloads").
+		SetTextAlign(tview.AlignLeft)
+
+	listDownload := tview.NewList().
```

Comment:
would be good if you can quickly change it to call dummy function that gets downloaded files, otherwise we can leave this be at the moment.

Reply:
for this PR can we just leave it like this? I will update it in the new PR. What do you think?
So, it keep the file changes in this PR keep small